python/interpret_community/mimic/mimic_explainer.py

```diff
@@ -313,6 +313,24 @@ def __init__(self, model, initialization_examples, explainable_model, explainabl
         self._original_eval_examples = None
         self._allow_all_transformations = allow_all_transformations
 
+    def _get_surrogate_model_predictions(self, evaluation_examples):
```

Comment:
this will be used in automl code, is that correct?  wondering if maybe we should just make it public and remove the "_", or are there cases it won't work?

Reply:
hmm, maybe it's fine to keep it as private for now and make it public later.  Not sure what all of the advantages for/against this would be.

Reply:
I think this will be a private method. I think we can move the metric computation between predictions of teacher model and surrogate model in MimicExplainer itself.